src/components/Chat/InputMessageV2.tsx

```diff
@@ -136,7 +137,8 @@ const InputMessageV2 = ({
   onToggleConfirm,
   type,
   isShowToggle = true,
-  messageDisable
+  messageDisable,
+  isAnonimityEnabled = true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default value for `isAnonimityEnabled` is set to `true`. This might not be the best approach as it assumes that anonymity is enabled by default. It would be better to have this value come from a configuration or user settings, ensuring that the default behavior aligns with the application's requirements and user preferences.
```diff
-   isAnonimityEnabled = true
+   isAnonimityEnabled = false // or some other logic based on user settings or app configuration
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->